src/index.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import Bookstore from './components/Bookstore';
 // import { Provider } from 'react-redux';
```

Comment:
- [ ] Please remove this commented code.

Reply:
Done